Tools/Scripts/libraries/webkitbugspy/webkitbugspy/github.py

```diff
@@ -111,7 +111,8 @@ def validater(username, access_token):
             expiration = response.headers.get('github-authentication-token-expiration', None)
             if expiration:
                 expiration = int(calendar.timegm(datetime.strptime(expiration, '%Y-%m-%d %H:%M:%S UTC').timetuple()))
-            if (expiration is None or expiration > time.time()) and response.status_code == 200 and response.json().get('login') == username:
+                expiring_soon = 12 * 60 * 60  # Consider a token expiring in the next 12 hours to be expired
+            if (expiration is None or expiring_soon > int(time.time())) and response.status_code == 200 and response.json().get('login') == username:
```

Comment:
expiring_soon is not a delta, so `expiring_soon > int(time.time())` is always going to be False.